invenio_communities/assets/semantic-ui/js/invenio_communities/subcommunity/new.js

```diff
@@ -150,163 +151,174 @@ class CommunityCreateForm extends Component {
-                            checked={_get(values, "access.visibility") === item.value}
-                            value={item.value}
-                            onChange={({ event, data, formikProps }) => {
-                              formikProps.form.setFieldValue(
-                                "access.visibility",
-                                item.value
-                              );
-                            }}
-                          />
-                          <label className="helptext">{item.helpText}</label>
-                        </React.Fragment>
-                      ))}
-                    </>
-                  )}
-                </Grid.Column>
-              </Grid.Row>
-              <Grid.Row>
-                <Grid.Column textAlign="center">
-                  <Button
-                    positive
-                    icon
-                    labelPosition="left"
-                    loading={isSubmitting}
-                    disabled={isSubmitting}
-                    type="button"
-                    onClick={(event) => handleSubmit(event)}
-                  >
-                    <Icon name="plus" />
-                    {hasCommunity
-                      ? i18next.t("Create request")
-                      : i18next.t("Create community")}
-                  </Button>
-                </Grid.Column>
-              </Grid.Row>
-            </Grid>
-          </Form>
-        )}
-      </Formik>
+                    )}
+                    {!hasCommunity && (
+                      <>
+                        <TextField
+                          required
+                          id="metadata.title"
+                          fluid
+                          fieldPath="metadata.title"
+                          // Prevent submitting before the value is updated:
+                          onKeyDown={(e) => {
+                            e.key === "Enter" && e.preventDefault();
+                          }}
+                          label={
+                            <FieldLabel
+                              htmlFor="metadata.title"
+                              icon="book"
+                              label={i18next.t("Community name")}
+                            />
+                          }
+                        />
+                        <IdentifierField formConfig={formConfig} />
+                      </>
+                    )}
+                    {!_isEmpty(customFields.ui) && (
+                      <CustomFields
+                        config={customFields.ui}
+                        templateLoaders={[
+                          (widget) => import(`@templates/custom_fields/${widget}.js`),
+                          (widget) => import(`react-invenio-forms`),
+                        ]}
+                        fieldPathPrefix="custom_fields"
+                      />
+                    )}
+                    {canCreateRestricted && (
```

Comment:
this is not part of the mockups for communities, even though it's part of the "new community" form. Therefore, I think this can be removed (the backend for subcommunities is not considering the access settings right now)


WDYT @slint